router/router.go

```diff
@@ -79,7 +79,13 @@ func NewFromConfig(filename string) (Router, error) {
 	return router, nil
 }
 
-func newFromConfigBytes(fileBytes []byte) (Router, error) {
+// NewFromConfigBytes constructs a Router using the configuration specified
+// as bytes typically read from a binary file. This allows us to
+// package kv routing yaml files into binaries like gearcmd.
+// The routing rules should be placed under the "routes" key on
+// the root-level map in the file. Validation is performed as described in
+// parse.go.
+func NewFromConfigBytes(fileBytes []byte) (Router, error) {
```

Comment:
I think you have to create a parallel to this function: https://github.com/Clever/kayvee-go/blob/master/logger/logger.go#L90-L94

Reply:
I will do that in the next commit since logger relies on gopkg v6

Reply:
ahh, coool